code/modules/mob/living/brain/brain_item.dm

```diff
@@ -279,6 +276,8 @@
 	//since these people will be dead M != usr
 
 	if(!target_has_brain)
+		if(!isipc(C) && zone != BODY_ZONE_HEAD)
+			return
```

Comment:
burda neyi engellemeye çalıştığını tam olarak anlayamadım galiba
normalde bu kod beyni olmayan vücutlara ameleyat gerekmeden beyin takılmasını sağlıyor. eğer amacın ipc için beynin kafaya takılmasını önlemekse ipcipc && zone != chest olması gerek

Reply:
beynin cheste takilabildigi bi tek ipc braini var, burda eger ipc braini cheste ayarlayip ipc olmayan bi adamin vucuduna koymaui engellemek istedim ama ipc beyninin attack procuna bunu eklesem daha iyi olucak. 

Yani yapmak istedigim sey suydu, ipc beyni eger istenirse modu degistirilip kafaya takilabilcek yada tekrar degistirip cheste takilabilcek, cheste takilirken ipc disindaki canlilara takilamasin diye bu kodu yazmistim

Reply:
bu dugundugum gibi deiglmis ya